media/pages/sk/ochrana-osobnych-udajov/page.html

```diff
@@ -43,7 +43,7 @@ <h3 id="identifikacne-udaje-1">
 </p>
 
 <p id="identifikacne-udaje-1-2">
-<span>1.2.</span>
+<strong class="chv-color-blue">1.2.</strong>
```

Comment:
Pozeram, ze je tam tej modrej teraz prilis vela. Asi bude lepsie nechat iba `strong` bez modrej farby. Rovnako ako si to spravil pri "a)" odrazkach. A tucnou modrou nechame iba nadpisy.

Reply:
:ok: